Description: The test assumes a wrong OOM score.
 LP is set to -1000 and so it fails on the builders.
 Skip it for now.
Author: Utkarsh Gupta <utkarsh.gupta@canonical.com>
Origin: vendor
Forwarded: https://github.com/containerd/containerd/issues/8219
Last-Update: 2023-03-07

--- a/sys/oom_linux_test.go
+++ b/sys/oom_linux_test.go
@@ -48,6 +48,7 @@
 }
 
 func TestSetNegativeOomScoreAdjustmentWhenUnprivilegedHasNoEffect(t *testing.T) {
+	t.Skip("skip this test as it fails on LP builders because initial oom score is set to -1000.")
 	if runningPrivileged() && !userns.RunningInUserNS() {
 		t.Skip("needs to be run as non-root or in user namespace")
 		return
